dashboard/src/components/SideMenu/SideMenu.tsx

```diff
@@ -71,7 +71,10 @@ const SideMenu = (): JSX.Element => {
       orientation="vertical"
     >
       <div className="w-full px-4">
-        <img src="kernelCI_logo.png" className="text-onSecondary size-14" />
+        <img
+          src="kernelci-logo-white.svg"
+          className="text-onSecondary max-w-[125px]"
```

Comment:
Do we need this `text-onSecondary` class ? 🤔

Reply:
Fixed!